jobs/build/promote/Jenkinsfile

```
@@ -210,45 +210,10 @@ node {
         
         stage("Check for Blocker Bugs") {
             group = "openshift-${major}.${minor}"
-            blocker_bugs = commonlib.shell(
-                returnStdout: true,
-                script: "${buildlib.ELLIOTT_BIN} -g ${group} find-bugs --mode blocker --report"
-            ).trim()
-
-            echo blocker_bugs
-            
-            found = true
-            try {
-                pattern = ~"Found ([0-9]+) bugs"
-                match = blocker_bugs =~ pattern
-                match.find()
-                found = (match[0][1] != '0')
-            } catch(ex) {
-                echo "Please check for blocker bug output"
-            }
             
-            if (found) {
-                msg = "Blocker bugs found! Prompting ARTist for confirmation"
-                echo msg
-                taskThread.say(msg)
-                commonlib.inputRequired(taskThread) {
-                    def resp = input(
-                        message: "Blocker Bugs found. Do not proceed until confirmation from ARTists/bug owners that it is okay to proceed. If the bugs are not blockers, coordinate to remove the blocker+ flag on the bug(s)",
-                        parameters: [
-                            booleanParam(
-                                defaultValue: false,
-                                description: "Check this checkbox only if you're sure you want to proceed with pending release blocker bugs. Ask #forum-release on slack if you are not sure.",
-                                name: 'IGNORE_BLOCKER_BUGS',
-                            ),
-                        ]
-                    )
-                    if (!resp) {
-                        currentBuild.result = 'ABORTED'
-                        error('Aborting because release blocker bugs found.')
-                    }
-                }
-            } else {
-                echo "No blocker bugs found. Proceeding"
+            commonlib.retryAbort("Waiting for Blocker Bugs to be resolved", taskThread, 
+                                    "Blocker Bugs found for release. Do not proceed until confirmation from ARTists/bug owners that it is okay to proceed. If the bugs are not blockers, coordinate to remove the blocker+ flag on the bug(s). RETRY when blocker(s) is finally resolved.") {
```

Rather than putting a lot of detail here I'd rather just refer to the doc.
```suggestion
                                    "Blocker Bugs found for release; do not proceed without resolving. See https://github.com/openshift/art-docs/blob/master/4.y.z-stream.md#handling-blocker-bugs") {
```